src/api/app/jobs/measurements_job.rb

```diff
@@ -5,36 +5,49 @@ def perform
     return unless CONFIG['amqp_options']
 
     RabbitmqBus.send_to_bus('metrics', "group count=#{Group.count}")
-    RabbitmqBus.send_to_bus('metrics', "user #{user_fields}")
+    RabbitmqBus.send_to_bus('metrics', "user #{measurements_to_fields(users_measurements)}")
+    RabbitmqBus.send_to_bus('metrics', "notification #{measurements_to_fields(notifications_measurements)}")
   end
 
   private
 
-  def user_fields
-    user_measures.map { |k, v| "#{k}=#{v}" }.join(',')
+  def measurements_to_fields(measurements)
+    measurements.map { |k, v| "#{k}=#{v}" }.join(',')
   end
 
-  def user_measures
+  def users_measurements
     {
       in_beta: User.in_beta.count,
       in_rollout: User.in_rollout.count,
       count: User.count
     }
-      .merge(role_fields)
-      .merge(state_fields)
+      .merge(roles_measurements)
+      .merge(states_measurements)
   end
 
-  def role_fields
+  def roles_measurements
     Role.global.pluck(:title).each_with_object({}) do |role_title, fields|
       fields[role_title.downcase.to_sym] = Role.find_by(title: role_title).users.count
       fields
     end
   end
 
-  def state_fields
+  def states_measurements
     User::STATES.each_with_object({}) do |state, fields|
       fields[state.to_sym] = User.where(state: state).count
       fields
     end
   end
+
+  def notifications_measurements
+    {
+      count: Notification.count,
+      for_web: Notification.for_web.count,
+      for_rss: Notification.for_rss.count,
+      read: Notification.where(delivered: true).count,
```

Comment:
To reuse the `NotificationsFinder` query object:

```suggestion
      read: NotificationsFinder.new.read.count,
```

Reply:
changed